### PR TITLE
Fix change detection in CLI when publish folder contains space

### PR DESCRIPTION
**Pull request by MartinRL**
### Summary & Motivation

On Windows, the publish folder for the CLI may contain spaces (due to a user account name with spaces), MSBuild will split the path using space and create multiple switches, resulting in the build error MSB1008. Example path: C:\Users\Martin Rosén-Lidholm\AppData\Local\PlatformPlatform. The fix is to put quotes around the path.

### Checklist

- [x] I have added a Label to the pull-request
- [x] I have added tests, and done manual regression tests
- [x] I have updated the documentation, if necessary

### CODE REVIEW

**Review by tjementum — CHANGES_REQUESTED** [review]
Awesome @MartinRL. Thank you for testing PlatformPlatform. I'm curious to hear your thoughts.

Only one small comment, as the commit message should be in title case (starting with an uppercase letter) and without emojis :)

I would be happy if you would force-push a commit, and then we will merge right away.

P.S.: I've added the "checklist from the pull-request template" to the bottom of the pull-request description.

**Review by tjementum — APPROVED** [review]

**Comment by tjementum** [comment]
@MartinRL Not sure exactly why, but it looks like you will have to sign the license/cla again. It could be because a force push rewrites history. As soon as you do, it will merge.

**Comment by MartinRL** [comment]
Seems like I can't sign the CLA again after the fact. I'll close and re-create the PR.

**Comment by tjementum** [comment]
I think might not trigger again. I will try to force merge.

**Pull request merged**
